pom.xml

```
@@ -175,8 +189,8 @@
                 <version>3.5.1</version>
                 <inherited>true</inherited>
                 <configuration>
-                    <source>8</source>
-                    <target>8</target>
+                    <source>17</source>
+                    <target>17</target>
```

Use ${java.version} here
```suggestion
                    <source>${java.version}</source>
                    <target>${java.version}</target>
```